nebula3/sclient/GraphStorageClient.py

```diff
@@ -225,6 +234,9 @@ def _scan_vertex(
         req.filter = where
         req.only_latest_version = only_latest_version
         req.enable_read_from_follower = enable_read_from_follower
+        req.username = self.user.encode('utf-8')
+        req.password = self.passwd.encode('utf-8')
+        req.need_authenticate = True
```

Comment:
why hard code true?

Reply:
need_authenticate is useful for storaged to distinguish the request source is graphd or clients. if the request source is clients, it will config true, and if the request source is graphd, it does not need to auth.